### PR TITLE
Main page branch request

### PR DESCRIPTION
**Pull request by kenderb**
This is our first pull request for the Youtube Clone.
This request includes HTML and CSS files also some images for the icons and buttons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        </li>
+        <li class="UploadAndSignContainer displayFlexbox">
+          <button class="upload"></button>
+          <button class="SignIn">Sign in</button>
+        </li>
+      </ul>
+    </nav>
+    <header class="popUp displayFlexbox whiteBGColor">
+      <h4 class="displayFlexbox">
+        <!--  <img src="" alt="" />-->
```
Comment (acushlakoncept) [inline index.html]: Remove unused comment

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello, @kenderb  and @AlejandroNo4  Great job! 👏  

#### First of all, congrats in working on your first Microverse project 🌟

You've done an awesome job so far 👌. But there are still some changes that are necessary for this project to be approved.
Let's quickly add some finishing touches and request another review.

- [ ] Your page needs to have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar, I see you already have two. 
icons_requirement

- [ ] Your HTML contains [validation errors](https://validator.w3.org/)


### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

File: index.html (hunk 1 above)
Comment (kenderb) [reply index.html]: Thanks, we added more changes and created more icons for the left side of the navbar.

**Review by arslanbisharat — COMMENTED** [review]
## Approved 
Thank you for making those changes requested by previous TSE. You can merge the branch and move forward with your curriculum. 
I am eager too see your next project 
Arslan Bisharat

**Comment by kenderb** [comment]
#Thanks!!

**Pull request merged**